### ISSUE TITLE
Some tips

### ISSUE THREAD

**Issue by allevo** [now-ims/now-fastify-datastore#1]
Hi!

I have made a little review on the code.
This is a simple plugin and this is amazing thing!

Only a little nit:

https://github.com/now-ims/now-fastify-datastore/blob/4741dc6c63387f2e357c81dd8dae888c15ad0b6a/index.js#L9-L11

At those lines, the `if` checks if the fastify instance has already a property called `ds`. In some situation this could be an unexpected behaviour.

Consider the following code

```js

const dataStorePlugin = require('@now-ims/fastify-datastore')
fastify
  .register(dataStorePlugin, { projectId: 'id1', keyFilename: '/path/to/keyfile1.json' })
  .register(dataStorePlugin, { projectId: 'id2', keyFilename: '/path/to/keyfile2.json' })
  .register(dataStorePlugin, { projectId: 'id3', keyFilename: '/path/to/keyfile3.json' })
```

Which is the behaviour you will expect?

From my point of view that code should raise an error. Fastify already manages that error for you when the code decorates the fastify instance twice.

If you are looking for a more complex solution to handle a lot of dataStore clients, please consider to see how [fastify-mongodb](https://github.com/fastify/fastify-mongodb) implements that feature

**Comment by puchesjr** [comment]
@allevo good point. thanks for the feedback. I will update the plugin to accommodate multiple connections.
